app/src/main/java/rightmesh/left/io/gpstracker/MainActivity.java

```diff
@@ -33,15 +35,18 @@
 
 public class MainActivity extends AppCompatActivity implements MeshStateListener {
     private static final String TAG = MainActivity.class.getCanonicalName();
-    private static final int MESH_PORT = 5001;
+    private static final int MESH_PORT = 1001;
```

Comment:
Because I don't see any app_key with 5001 port in dev portal, and I can't add a new key for that port with this message `Error! Mesh port has already been taken. Please pick another port number`

Reply:
You probably need the person who created the key to give you access to it

Reply:
@conbrad

Reply:
Added everyone under the test app with this key. You should be able to use it now.